crates/l2/contracts/src/example/L2ERC20.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.29;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../l2/interfaces/IERC20L2.sol";
+
+/// @title OnChainProposer contract.
+/// @author LambdaClass
+contract TestTokenL2 is ERC20, IERC20L2 {
+    address public L1_TOKEN = address(0);
+    address public constant BRIDGE =  0x000000000000000000000000000000000000FFff;
```

Comment:
Should we put the bridge address on the interface so token operators don't mess up?